docs/reference/data-frames/apis/put-transform.asciidoc

```diff
@@ -15,7 +15,13 @@ Instantiates a {dataframe-transform}.
 
 `PUT _data_frame/transforms/<data_frame_transform_id>`
 
-//===== Description
+===== Description
+
+IMPORTANT:  You must use {kib} or this API to create a {dataframe-transform}.
+            Do not put a {dataframe-transform} directly to the
+            `.data-frame-internal-1` index using the Elasticsearch index API.
+            If {es} {security-features} are enabled, do not give users `write`
+            privileges on the `.data-frame-internal-1` index.
```

Comment:
Maybe "the `.data-frame-internal-1` index" -> "any `.data-frame-internal*` indices".